acronym/acronym.rb

```diff
@@ -0,0 +1,28 @@
+=begin
+Write your code for the 'Acronym' exercise in this file. Make the tests in
+`acronym_test.rb` pass.
+
+To get started with TDD, see the `README.md` file in your
+`ruby/acronym` directory.
+=end
+
+module Acronym 
+    def self.abbreviate(frase)
+        #frase es un string ejem "compu mundo-hiper mega red"
+        withoutdash = frase.gsub(/-/, " ")
+        acronimo = []
```

Comment:
For consistency, use english to name all variables, functions, classes, and even in comments. Use spanish only when it's hard to do a translation. For example, concepts like "DIAN", "IVA", etc are hard to translate to english.